### PR TITLE
Trigger translations

### PR DESCRIPTION
**Pull request by CDRussell**
Task/Issue URL: https://app.asana.com/0/0/1204100098948920/f

### Description
Translations for new copy

### CODE REVIEW

**Comment by CDRussell** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2907** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2907" target="_blank">Graphite</a> 
    * **PR #2908** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2908" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2908?utm_source=stack-comment).

**Comment by CDRussell** [comment]
@cmonfortep, can I ask you to review the mac string changes? (I've checked all the autofill ones)

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -618,7 +618,7 @@
 
     <!-- MacOs Settings -->
     <string name="macos_settings_description">Navega de forma privada con nuestra aplicación para Mac</string>
-    <string name="macos_settings_title">Aplicación de escritorio DuckDuckGo</string>
+    <string name="macos_settings_title">Aplicación DuckDuckGo para Mac</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:621]: This copy is already in develop...so not sure why it appears in the diff.

**Review by cmonfortep — APPROVED** [review]
LGTM, as commented the macOS strings shouldn't be part of the diff, as those values are already in develop. Not sure why you see them here, but it's the expected value.

**Pull request merged**